packages/dev/core/src/Materials/material.ts

```diff
@@ -321,7 +321,7 @@ export class Material implements IAnimatable, IClipPlanesHolder {
 
         // Only call dirty when there is a state change (no alpha / alpha)
         if (oldValue === 1 || value === 1) {
-            this.markAsDirty(Material.MiscDirtyFlag);
+            this.markAsDirty(Material.MiscDirtyFlag + Material.PrePassDirtyFlag);

```

Comment:
shouldn't that be `&`?

Reply:
No, I want to add the PrePass dirty flag in addition to the already existing Misc dirty flag (MiscDirtyFlag is not a "catch all" flag like 0xFF, it's a normal flag like TextureDirty or PrePassDirty). For strict correctness, I should have used `|`.